bootstrap/azure/config-default.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Keeping atlas variable without prefix as it's been shared by consul and tf at the moment.
+export ATLAS_TOKEN=${ATLAS_TOKEN:?"Need to set ATLAS_TOKEN non-empty"}
+export ATLAS_INFRASTRUCTURE=${ATLAS_INFRASTRUCTURE:-capgemini/apollo}
+
+export TF_VAR_azure_settings_file=${TF_VAR_azure_settings_file:?"Need to set TF_VAR_azure_settings_file non-empty"}
+export TF_VAR_user=${TF_VAR_user:?"Need to set User non-empty"}
+export TF_VAR_username=${TF_VAR_username:?"Need to set TF_VAR_username non-empty"}
+
+export TF_VAR_key_file=${TF_VAR_key_file:-$HOME/.ssh/apollo_azure_rsa.pfx}
+export TF_VAR_key_name=${TF_VAR_key_name:-apollo}
+
+# Overrides default folder in Terraform.py inventory.
+export TF_VAR_STATE_ROOT="${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
+
+export ANSIBLE_SSH_ARGS="-F ${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}/ssh.config -q"
+
+# Terraform mappings needs to be statically passed as -var parameters
+# so no really needed to export them. Exporting for consitency.
+export TF_VAR_atlas_artifact_master=${TF_VAR_atlas_artifact_master:-apollo-ubuntu-14.04-amd64-1446553949}
```

Comment:
we use azure images which are kept on azure storage here but I use the same variables for that because they are hardcoded in common.sh file in terraform_apply function so if I remove them I will get an error. Probably we could use here a default option to avoid the problem.